contracts/ExampleTrustcoin2.sol

```diff
@@ -64,87 +58,68 @@ contract ExampleTrustcoin2 is ERC20TokenInterface, SafeMath {
   }
 
   // See ERC20
-  function balanceOf(address _owner) constant returns (uint balance) {
+  function balanceOf(address _owner) constant external returns (uint balance) {
     return balances[_owner];
   }
 
   // See ERC20
-  function approve(address _spender, uint _value) returns (bool success) {
+  function approve(address _spender, uint _value) external returns (bool success) {
     allowed[msg.sender][_spender] = _value;
     Approval(msg.sender, _spender, _value);
     return true;
   }
 
   // See ERC20
-  function allowance(address _owner, address _spender) constant returns (uint remaining) {
+  function allowance(address _owner, address _spender) constant external returns (uint remaining) {
     return allowed[_owner][_spender];
   }
 
   //
   //  Migration methods
   //
 
-  /**
-   *  Changes the owner for the migration behaviour
-   *  @param _master Address of the migration controller
-   */
-  function changeMigrationMaster(address _master) onlyFromMigrationMaster external {
+  // See MigratedTokenInterface
+  function migrateOldTokens(address _from, uint256 _value) external {
```

Comment:
I'm missing something here. Either this is the method users call to migrate[1] and then it should check with the old contract to discard the tokens, or this method is called by the old contract, in which case I don't see where we chat that.



[1] See comment in MigratedTokenInterface about which contract users initiate the migration from.